sal/class.json-api-site-jetpack.php

```diff
@@ -108,6 +108,10 @@ function get_frame_nonce() {
 		return false;
 	}
 
+	function get_frame_nonce_site_only() {
+		return false;
+	}
+
```

Comment:
I know that none of the other functions are documented, but could we add a short explanatory comment here on why it always returns `false`? having one above in `get_frame_nonce()` would also be helpful and aid the casual passer-by as to why these two are different. I'm going to toss an example out to illustrate my point but I assume that everything I say in my example is wrong.

```php
function has_wordads() {
	return Jetpack::is_module_active( 'wordads' );
}

/**
 * Prevents creating an iframe nonce for connected sites (I have no idea what this is here for)
 *
 * **Note**: On WordPress.com there is no nonce because tigers
 */
function get_frame_nonce() {
	return false;
}

/**
 * Prevents creating a site-only iframe nonce for connected sites
 *
 * The site-only nonce is used in the iframe'd block editor
 * and exists because of user authentication (please write something meaningful here instead)
 * @see Some Reference To Documentation On This
 *
 * **Note**: On WordPress.com there is no nonce because bears
 */
function get_frame_nonce_site_only() {
	return false;
}
```